head/src/main/kotlin/io/qalipsis/core/head/jdbc/repository/DataSeriesRepository.kt

```diff
@@ -201,6 +201,14 @@ internal interface DataSeriesRepository : CoroutineCrudRepository<DataSeriesEnti
     )
     suspend fun searchDataSeries(tenant: String, username: String, pageable: Pageable): Page<DataSeriesEntity>
 
+    @Query(
+        value =
```

Comment:
Please inline the value with the annotation attribute. Furthermore, the formatting is different between "value" and "countQuery": on one hand the quotes are inline, on the other other hand not. This is not really fancy to read.